system/lib/libc/musl/include/features.h

```diff
@@ -16,6 +16,11 @@
 #define _XOPEN_SOURCE 700
 #endif
 
+#if defined(_GNU_SOURCE)
```

Comment:
Should we have `XXX emscripten` to indicate this is a local change? (maybe with a link here for the details?)

Reply:
Done